### PR TITLE
systemd 251

### PR DESCRIPTION
**Pull request by SMillerDev**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Okay, that lib exists in `lib/systemd/libsystemd-shared-251.so`, I'm not sure why linkage is complaining.

**Comment by carlocab** [comment]
```ruby
ENV.append "LDFLAGS", "-Wl,-rpath,#{lib}/systemd"
```

File: Formula/systemd.rb [hunk 1]
```diff
@@ -34,35 +34,25 @@ class Systemd < Formula
...
+    args = *std_meson_args + %W[
       --sysconfdir=#{etc}
       --localstatedir=#{var}
       -Drootprefix=#{prefix}
-      -Dsysvinit-path=#{prefix}/etc/init.d
-      -Dsysvrcnd-path=#{prefix}/etc/rc.d
-      -Dpamconfdir=#{prefix}/etc/pam.d
+      -Dsysvinit-path=#{etc}/init.d
+      -Dsysvrcnd-path=#{etc}/rc.d
+      -Dpamconfdir=#{etc}/pam.d
```
Comment (carlocab) [inline Formula/systemd.rb:46]: I think @iMichka put these in `prefix/"etc"` on purpose, though I don't know why. They look like configuration directories, though, which definitely belong in `etc`.

Comment (iMichka) [reply Formula/systemd.rb:47]: There are some explanations here https://github.com/Homebrew/linuxbrew-core/issues/5408 and https://github.com/Homebrew/linuxbrew-core/pull/5446, these paths were added to fix a compilation issue found in 2018.

Comment (SMillerDev) [reply Formula/systemd.rb:47]: It looks like there it just extended the existing `etc` references.

Comment (carlocab) [reply Formula/systemd.rb:47]: Well, it seems to compile fine now. And the dependent failure is definitely unrelated. (Still worth investigating tho)

**Review by carlocab — APPROVED** [review]

File: Formula/systemd.rb (hunk 1 above)
Comment (danielnachun) [reply Formula/systemd.rb:47]: I will submit a fix for the dependent failure in `pyside@2`, which is indeed completely unrelated.

**Review by danielnachun — APPROVED** [review]

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**